tests/share/test_janitor.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+from unittest import mock
+
+from share.janitor.tasks import rawdata_janitor
+
+from tests import factories
+
+
+@pytest.mark.django_db
+class TestRawDataJanitor:
+
+    @pytest.fixture(autouse=True)
+    def mock_transform(self, monkeypatch):
+        mock_transform = mock.Mock()
+        monkeypatch.setattr('share.tasks.transform.apply', mock_transform)
+        return mock_transform
+
+    def test_empty(self, mock_transform):
+        rawdata_janitor()
+
+        assert mock_transform.called is False
+
+    def test_unprocessed_data(self, mock_transform):
```

Comment:
Maybe should also have a test where only some, not all, raw data are unprocessed, just in case.